src/cow-react/modules/swap/hooks/useSwapRawState.ts

```diff
@@ -15,22 +15,19 @@ export function useSwapRawState(): TradeRawState {
   }
 }
 export function useUpdateSwapRawState(): (update: Partial<TradeRawState>) => void {
-  const swapState = useSwapState()
   const dispatch = useAppDispatch()
```

Comment:
This state was stale, not being updated by the time the callback on line `23` was executed.
Resulting in the old overwrite the new state